lib/kafo/hook_context.rb

```diff
@@ -90,5 +91,10 @@ def get_custom_config(key)
     def store_custom_config(key, value)
       self.kafo.config.set_custom(key, value)
     end
+
+    # Return the path to the current scenario
+    def scenario
```

Comment:
Should his be ```scenario_path``` and should we add a parsed version as well?

Reply:
sounds good to me, since this is sort of an API for installers, we should make it right on first attempt :-)